src/selenium/driver.cr

```diff
@@ -41,8 +41,13 @@ abstract class Selenium::Driver
   def create_session(capabilities, retry = true) : Session
     parameters = {capabilities: {alwaysMatch: capabilities}}
     data = command_handler.execute(:new_session, parameters: parameters)
+    session_id = data.dig?("value", "sessionId").try(&.as_s?)
 
-    Session.new(http_client, command_handler, data.dig("value", "sessionId").as_s)
+    if session_id.nil?
+      raise InvalidSessionError.new(data.dig("value", "message").as_s)
+    end
```

Comment:
I added this because the original error was "Missing key sessionId" which wasn't helpful. Getting the message out here was actually saying "DevToolsActivePort file doesn't exist". Looking that up led me to adding the remote-debugging-port.